website/content/docs/connect/gateways/api-gateway/index.mdx

```diff
@@ -1,28 +1,61 @@
 ---
 layout: docs
-page_title: API Gateways Overview
-description: API gateways are objects in Consul that enable ingress requests to services in your service mesh. Learn about API gateways for VMs in this overview.
+page_title: API gateways overview
+description: API gateways provide an ingress point for service mesh traffic. Learn how API gateways add listeners for external traffic and route HTTP requests to services in the mesh.
 ---
 
-# API gateway overview
+# API gateways overview
 
-API gateways enable external network clients to access applications and services
-running in a Consul datacenter. This type of network traffic is commonly
-called _north-south_ network traffic because it refers to the flow of
-data into and out of a specific environment. API gateways can also forward
-requests from clients to specific destinations based on path or request
-protocol.
+This topic provides overview information about API gateways in Consul. API gateways enable external network clients to access applications and services running in a Consul datacenter. Consul API gateways can also forward requests from clients to specific destinations based on path or request protocol.
+
+## API gateway use cases
```

Comment:
Does this also need to be gateway(s)?

Reply:
I suppose it could, but the ideas in this phrase was to use "API gateway" and "use" as a compound modifier for "cases" so that the meaning is something like "Use an API gateway in these cases:"